client/client_test.go

```diff
@@ -42,6 +43,24 @@ func TestClient_Cred_PathMissing(t *testing.T) {
 	}
 }
 
+func TestClient_Cred_EnvOverride_ExplicitPath(t *testing.T) {
+	configFilePath := "testdata/certificate_config.json"
+	os.Setenv("GOOGLE_API_CERTIFICATE_CONFIG", "testdata/certificiate_config_missing_path.json")
+	_, err := Cred(configFilePath)
```

Comment:
I think we will want to inspect the cred object to make sure it used the correct path

Reply:
Okay, I will look more into how to access information from the Cred object

Reply:
See what you find in the wrapped `Cmd` struct!

Reply:
Liam sorry I overlooked this scenario in our 1:1, I think there is one other thing you can do without inspecting the Cred object to make sure your override is being triggered. You can point the override to a "broken" json file (ex missing a brace or something) and verify you get the error expected - (since otherwise you can't tell whether the test passed because there is no default config on the machine). If you can find an easy way to inspect the Cred that would work too, and yes feel free to add new test data files.